data/equiv/basic.lean

```diff
@@ -177,11 +177,23 @@ def equiv_pempty (h : α → false) : α ≃ pempty :=
 def false_equiv_pempty : false ≃ pempty :=
 equiv_pempty _root_.id
 
+def empty_equiv_pempty : empty ≃ pempty :=
+equiv_pempty $ empty.rec _
+
+def pempty_equiv_empty : pempty ≃ empty :=
```

Comment:
Are both `empty_equiv_pempty` and `pempty_equiv_empty ` needed since `≃` is symmetric?

Reply:
I think having both would be more convenient.

Reply:
Can't you just write `empty_equiv_pempty.symm` for `pempty_equiv_empty`?

Reply:
@digama0 what do you think?

Reply:
We don't usually have both symmetric versions of a definition like this. It is easy enough to use `symm`, and you will see this used in a few other places on this page.